src/rpc/rpc_reward_api.py

```diff
@@ -73,7 +75,7 @@ def get_rewards_for_cycle_map(self, cycle, expected_rewards=False):
                 total_block_reward = nb_blocks * self.block_reward
                 total_endorsement_reward = nb_endorsements * self.endorsement_reward
 
-                logger.info("Ideal rewards for cycle {:d}, {:,} block rewards ({:d} blocks), {:,} endorsing rewards ({:d} slots)".format(
+                logger.info("Estimated rewards for cycle {:d}, {:,} block rewards ({:d} blocks), {:,} endorsing rewards ({:d} slots)".format(
                             cycle, total_block_reward, nb_blocks, total_endorsement_reward, nb_endorsements))
 
                 reward_data["total_rewards"] = total_block_reward + total_endorsement_reward
```

Comment:
I am confused here. This whole code block is inside the `if rewards_type.isEstimated():` ???

Reply:
Yes, because this whole code block is for estimated reward calculation. Ideal/actual rewards use another code block in the `else` case below.